src/client/stories/index.js

```diff
@@ -20,6 +23,20 @@ storiesOf("Button", module)
 const MapStories = storiesOf("Map", module).add("MapComponent", MapStory);
 MapStories.addDecorator(withKnobs);
 
+const NavigationStories = storiesOf("Navigation", module).add(
+  "ButtonComponent",
+  NavButtonStory
+);
+NavigationStories.addDecorator(withKnobs);
+
+const VoyageStories = storiesOf("Start Voyage", module)
+  .add("StartVoyage", StartVoyageStory)
+  .add("DateTimeComponent", DateTimeStory)
+  .add("InputFieldNumberComponent", InputFieldNumberStory)
+  .add("DropdownComponent", DropdownStory)
+  .add("CheckboxComponent", CheckboxStory);
+VoyageStories.addDecorator(withKnobs);
```

Comment:
Let's get the form elements in a separate story?

```suggestion
const VoyageStories = storiesOf("Start Voyage", module)
  .add("StartVoyage", StartVoyageStory)
VoyageStories.addDecorator(withKnobs);

const FormStories = storiesOf("Forms elements", module)
  .add("DateTimeComponent", DateTimeStory)
  .add("InputFieldNumberComponent", InputFieldNumberStory)
  .add("DropdownComponent", DropdownStory)
  .add("CheckboxComponent", CheckboxStory);
FormStories.addDecorator(withKnobs);
```

Reply:
Changes have been incorporated with all form elements moved to a separate story.